neoexchange/core/models/blocks.py

```diff
@@ -249,6 +250,16 @@ def num_spectra_complete(self):
     def num_candidates(self):
         return Candidate.objects.filter(block=self.id).count()
 
+    def where_observed(self):
+        where_observed=''
+        if self.num_observed is not None:
```

Comment:
Is there a logical reason for using `if self.num_observed is not None` instead of `if self.num_observed`? Do you want to explicitly test for a `None` value or would `False` or `0` also be acceptible?

Reply:
> Sorry it took a while to get around to this. It was less controversial than I had first thought!

I probably should have done a better job of communicating the smallness of the change in the PR description. Good point about the specificity of the test; I will change that, merge and deploy on Monday.